transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_contact_update.sql

```diff
@@ -1,37 +1,66 @@
-{{config({
-    "schema": "hightouch",
+{{ config(
+    { "schema": "hightouch",
     "materialized": "view",
-    "tags":["hourly","blapi"]
-  })
-}}
+    "tags" :["hourly","blapi"] }
+) }}
 -- lead or contact already exists
 WITH existing_lead AS (
+
     SELECT
-        lead.id,
-        lead.email,
-        lead.ownerid
-    FROM {{ ref('lead') }}
-    WHERE converteddate IS NULL
-    QUALIFY ROW_NUMBER() OVER (PARTITION BY lead.email ORDER BY lead.createddate DESC) = 1
-), freemium_contacts_to_sync as (
+        LEAD.id,
+        LEAD.email,
+        CASE
+            WHEN SUBSTR(
+                LEAD.ownerid,
+                0,
+                3
+            ) = '00G' THEN NULL
+            ELSE LEAD.ownerid
+        END AS ownerid
+    FROM
+        {{ ref('lead') }}
+    WHERE
+        converteddate IS NULL qualify ROW_NUMBER() over (
+            PARTITION BY LEAD.email
+            ORDER BY
+                LEAD.createddate DESC
+        ) = 1
+),
+freemium_contacts_to_sync AS (
     SELECT
         customers_with_cloud_paid_subs.email,
         customers_with_cloud_paid_subs.first_name,
-        COALESCE(customers_with_cloud_paid_subs.last_name, customers_with_cloud_paid_subs.email) AS contact_last_name,
+        COALESCE(
+            customers_with_cloud_paid_subs.last_name,
+            customers_with_cloud_paid_subs.email
+        ) AS contact_last_name,
         existing_lead.id AS duplicate_lead_id,
         -- existing_lead.email AS duplicate_lead_email,
         customers_with_cloud_paid_subs.account_external_id,
-        COALESCE(existing_lead.ownerid, '0053p0000064nt8AAA') AS ownerid,
+        COALESCE(
+            existing_lead.ownerid,
+            '0053p0000064nt8AAA'
```

Comment:
Perhaps move `0053p0000064nt8AAA` to a project variable or macro so that it's managed and documented in a central place.